app/mailers/spool_submissions_report_mailer.rb

```diff
@@ -16,7 +16,8 @@ def build(report_file)
     opt = {}
     opt[:to] =
       if FeatureFlipper.staging_email?
-        'lihan@adhocteam.us'
+        ['lihan@adhocteam.us', 'akulkarni@meetveracity.com',
+         'Hoffmaster_David@bah.com', 'Turner_Desiree@bah.com', 'Delli-Gatti_Michael@bah.com']
```

Comment:
Maybe extract  this to a `STAGING_RECIPIENTS` constant

Reply:
thanks. change done to extract into a constant